contents/blog/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,134 @@
+
+For example, we've redesigned this entire website ~5 times.
+
+## Hiring is key
+
+I think 90%+ of an organization's problems are solved by hiring right – see [what we've learned about hiring](https://newsletter.posthog.com/p/everything-weve-learned-about-hiring).
+
+We have found it invaluable to pay people during our hiring process to do some actual work. We've refined this down to a one day task, which gives us a good sense of how much someone can get done and the quality of their actual code.
+
+We tend to hire people with more experience, who we find tend to work better with more autonomy. I think this is the magic combo when you're an all remote company, though we're working on ways to get better at growing people who are earlier in their careers.
+
+## Small teams
+
+I went to a superb talk by [Jeff Lawson](https://www.linkedin.com/in/jeffiel), who runs [Twilio](https://twilio.com/). He described how the greatest innovation at [Amazon](https://www.amazon.com/) was that it felt like thousands of small startups. Simply speaking, a startup gets more done per person than a big corporate.
+
+PostHog works like a group of startups with lots of [small teams](/handbook/company/small-teams). Each of our products has a team of up to 6 who can ship with minimal interference from the rest of the company. We hire a lot of former technical founders to help here.
+
+## Nearly everyone is an engineer
+
+It's simple, but if you have lots of spending and focus outside of engineering, you won't get as much built.
+
+At the time of writing, [we are 37 people](../team), only 11 of whom are in Go To Market or Operations roles (and 6 of those 11 have at least some engineering in their background).
+
+We don't do outbound sales, our [marketing team](/handbook/small-teams/marketing) is very small, we only have one product manager, our [exec team](/handbook/small-teams/exec) only has 3 people. These things are all by design (and were possible because of our market - the majority of engineers generally want docs and to try something instead of demos and salespeople to deal with).
+
+## No meeting Tuesdays/Thursdays
+
+We want a culture where people can deep work. We push back on unnecessary meetings (and yes, there are _definitely_ some that are necessary, we have learned - often painfully).
+
+## We are a late mover
+
+I've had lots of founders tell me that they can't pursue idea X because some other startups are tackling it. This makes me laugh because we simply won't build a new product unless there is a $1bn+ competitor already offering it. We _love_ competition. It shows you that there will be demand. The magic in our platform (after an individual product is working well for our users) is then integrating everything together.
```

Comment:
```suggestion
I've had lots of founders tell me that they can't pursue idea X because some other startups are tackling it. 

This makes me laugh because we simply won't build a new product unless there is a $1bn+ competitor already offering it. We _love_ competition. It shows you there's demand. 

The magic in our platform (after an individual product is working well for our users) is then integrating everything together.
```